modules/cluster/main.tf

```
@@ -85,6 +96,15 @@ resource "aws_iam_role_policy_attachment" "attach_worker_policy" {
   )
 }
 
+resource "aws_cloudwatch_log_group" "eks" {
+  count = var.manage_log_group ? 1 : 0
+
+  name              = "/aws/eks/${var.name}/cluster"
+  retention_in_days = var.log_group_retention
+
+  tags = local.merged_tags
```

Optional, but I would not separate these lines
```suggestion
  retention_in_days = var.log_group_retention
  tags              = local.merged_tags
```

---

I did this as common standards for resources would put `tags` last (as it's typically a map) after other inputs but before things like `depends_on`, `lifecycle`, etc.